tensorboard/webapp/routes/dashboard_deeplink_provider_test.ts

```diff
@@ -391,6 +391,30 @@ describe('core deeplink provider', () => {
     ]);
   });
 
+  it('dedupes unknown query params with feature flags', () => {
+    store.overrideSelector(selectors.getOverriddenFeatureFlags, {
+      enabledExperimentalPlugins: ['foo', 'bar', 'baz'],
+      showFlags: '',
+    });
+    store.overrideSelector(selectors.getUnknownQueryParams, {
+      unknown1: 'unknown1_value',
+      unknown2: 'unknown2_value',
+      // Should be ignored since it is also a feature flag query param.
+      experimentalPlugin:
+        'this_is_known_but_has_different_value_for_some_reason',
```

Comment:
Curious: do we know that we would want to use the value set in the "known" parameters and ignore the other one? Or is it the case that we can't know, but this is just a simple way of deduping? Or perhaps, we know in some scenarios, but not all of them?